src/current/v23.2/migration-strategy-live-migration.md

```diff
@@ -0,0 +1,95 @@
+
+A live migration consists of the following distinct phases:
+
+- The initial phase where the source database is the *source of truth*: the database used by the application to read and write data, and the source for replicating the data to CockroachDB, the target database.
+- The cutover phase where CockroachDB is made the source of truth and the source database is the secondary database used for failover.
+- The final phase where the source database is removed. The source database is no longer available as a replication target, or a failover database.
+
+There are many possible approaches to performing a live migration. In this topic, we describe two example approaches that have been successful with CockroachDB: consistent cutover, and immediate cutover.
+
+The consistent cutover approach uses only one database that is writable and consistently readable at the application level, with the data replicated on the back end by another service. When using the consistent cutover approach, one database is used by the application at any point in time. During the cutover phase, any current application transactions that use the source database are allowed to complete, and the replication service is allowed to catch up. After this pause, new application sessions are allowed to start using the target database, and the replication service replicates data from the target database to the source database. If any problems arise during the cutover phase and you have to revert back to the initial phase, the same process repeats, moving from the target database back to the source database.
+
+With the immediate cutover approach, the application begins writing to the target database with no pause. This approach can lead to incorrect or inconsistent data between the two databases if there were any active application transactions at the time of the cutover, and may require work after cutover to correct any unacceptable data inconsistencies. Cockroach Labs does not recommend the immediate cutover approach with the live migration strategy.
+
+Cockroach Labs recommends using the consistent cutover approach when using the live migration strategy. Your data remains more consistent and downtime is minimized. However, if zero downtime is truly required, the immediate cutover approach may be used at the risk of some data inconsistencies. Third-party migration tools like [AWS DMS]({% link {{ page.version.version }}/aws-dms.md %}), [Qlik]({% link {{ page.version.version }}/qlik.md %}), and [Striim]({% link {{ page.version.version }}/striim.md %}) perform the initial migration into CockroachDB, and replicate ongoing changes. CockroachDB [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}) allow for replication to the source database after cutover, simplifying the service architecture.
```

Comment:
Probably better to just reference the table of supported tools to avoid future updates https://www.cockroachlabs.com/docs/v23.1/third-party-database-tools#data-migration-tools

Reply:
Done.